subsys/testsuite/ztest/include/ztest_test.h

```diff
@@ -1,302 +1,32 @@
 /*
  * Copyright (c) 2016 Intel Corporation
+ * Copyright (c) 2021 Google LLC
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/**
- * @file
- *
- * @brief Zephyr testing framework _test.
- */
-
-#ifndef ZEPHYR_TESTSUITE_ZTEST_TEST_H_
-#define ZEPHYR_TESTSUITE_ZTEST_TEST_H_
+#ifndef ZEPHYR_TESTSUITE_INCLUDE_ZTEST_TEST_H_
+#define ZEPHYR_TESTSUITE_INCLUDE_ZTEST_TEST_H_
 
-#include <app_memory/app_memdomain.h>
-#include <init.h>
-#include <stdbool.h>
+#ifdef CONFIG_ZTEST_NEW_API
+#include <ztest_test_new.h>
+#else
+#include <ztest_test_deprecated.h>
+#endif /* !CONFIG_ZTEST_NEW_API */
```

Comment:
nit: Should the "!" be here given it's not an "ifndef" ?

Reply:
It's on the else, so I assumed so...